### PR TITLE
sysstat: new formula (Linux-only)

### PR DESCRIPTION
**Pull request by danielnachun**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by danielnachun** [comment]
I guess this also depends on https://github.com/Homebrew/homebrew-core/pull/76058.  I'm not entirely sure why this is coming up, as `sysstat` has no dependencies and `brew uses --formula --include-build --include-test --recursive sysstat` returns nothing on my test system.

**Comment by Bo98** [comment]
https://github.com/Homebrew/homebrew-core/commit/222182b6744b591e09b16253e6f64dc6e2b5cadc was merged with a dependency on `lm-sensors` but `lm-sensors` hasn't been merged so homebrew-core operations on Linux won't work right now.

**Comment by danielnachun** [comment]
Whoops, well that was an oversight.  Luckily it is easily fixed!  `lm-sensors` seems to work fine, the last run failed because of a typo.

File: Formula/sysstat.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  url "https://github.com/sysstat/sysstat/archive/v12.4.0.tar.gz"
+  sha256 "1962ed04832d798f5f7e787b9b4caa8b48fe935e854eef5528586b65f3cdd993"
+  license "GPL-2.0-or-later"
+  head "https://github.com/sysstat/sysstat.git"
+
+  depends_on :linux
+
+  def install
+    system "./configure",
+      "--disable-file-attr", # Fix install: cannot change ownership
```
Comment (MikeMcQuaid) [inline Formula/sysstat.rb:13]: This indentation feels a bit weird. I'd suggest aligning with the first argument on line above.

**Comment by danielnachun** [comment]
I fixed the indentation and wrote a better test which actually uses some of the functionality of a couple of the binaries built by the formula.

**Review by SMillerDev — APPROVED** [review]

**Review by SMillerDev — APPROVED** [review]

**Review by MikeMcQuaid — APPROVED** [review]

**Pull request merged**